package.json

```diff
@@ -28,17 +28,21 @@
 	"homepage": "https://github.com/seznam/IMA.js-server",
 	"devDependencies": {
 		"express": "^4.14.0",
-		"gulp": "^3.9.1",
+		"gulp": "github:gulpjs/gulp#3f5aba2",
 		"gulp-jasmine": "^2.4.2",
 		"gulp-plumber": "^1.1.0",
 		"jasmine": "^2.5.3",
 		"superagent": "^3.3.2"
 	},
 	"dependencies": {
-		"async-each": "1.0.1",
-		"highlight.js": "9.9.0",
-		"ima-helpers": "0.14.0-rc.1",
-		"stack-trace": "0.0.9",
-		"winston": "2.3.0"
+		"async-each": "^1.0.1",
```

Comment:
That would not freeze the deep dependencies. We agreed to using shrinkwrap in project to achieve better stability, remember? Leaving the versions like this will also make updating the dependencies less of an issue.

We may add a shrinkwrap file to the project skeleton if you'd like.